crates/forge/tests/cli/build.rs

```diff
@@ -51,6 +51,21 @@ forgetest_init!(build_sizes_no_forge_std, |prj, cmd| {
     assert!(stdout.contains("Counter"), "\n{stdout}");
 });
 
+// tests build output is as expected in zksync mode
+forgetest_init!(test_zk_dbg_build_sizes, |prj, cmd| {
+    use std::process::Command;
+    let output = Command::new("sh")
+        .arg("-c")
+        .arg("ls -l /home/runner/.zksync/zksolc-linux-amd64-musl-v1.5.1")
+        .output()
+        .expect("failed to execute process");
+    println!("{output:?}");
```

Comment:
Do we want to do this check here or just let foundry do it? How's this portable with non-ci environments?

Reply:
see above